assets/js/components/module/ModuleApp.js

```diff
@@ -68,20 +68,24 @@ function ModuleApp( { moduleSlug } ) {
 				{ moduleConnected && <DateRangeSelector /> }
 			</Header>
 			<Alert module={ moduleSlug } />
-			{ shouldRenderWidget &&
-				<Fragment>
-					<WidgetContextRenderer
-						slug={ screenWidgetContext }
-						className={ classNames( [
-							'googlesitekit-module-page',
-							`googlesitekit-module-page--${ moduleSlug }`,
-						] ) }
-						Header={ getModuleHeader }
-						Footer={ ModuleFooter }
-					/>
-				</Fragment>
+			{ shouldRenderWidget
+				? (
+					<Fragment>
+						<WidgetContextRenderer
+							slug={ screenWidgetContext }
+							className={ classNames( [
+								'googlesitekit-module-page',
+								`googlesitekit-module-page--${ moduleSlug }`,
+							] ) }
+							Header={ getModuleHeader }
+							Footer={ ModuleFooter }
+						/>
+					</Fragment>
```

Comment:
I know this is from before, but I don't think the wrapping `Fragment` is necessary here. Would you please take a look and see? If it isn't needed, let's remove it 👍